eth_tester/backends/pyevm/main.py

```diff
@@ -287,7 +305,33 @@ def __init__(self):
                 "`PyEVMBackend` requires py-evm to be installed and importable. "
                 "Please install the `py-evm` library."
             )
-        self.reset_to_genesis()
+
+        self.account_keys = None  # set below
+        accounts = len(genesis_state) if genesis_state else None
+        self.reset_to_genesis(genesis_parameters, genesis_state, accounts)
+
+    #
+    # Genesis
+    #
+
+    @staticmethod
+    def generate_genesis_params(overrides=None):
+        return get_default_genesis_params(overrides=overrides)
+
+    @staticmethod
+    def generate_genesis_state(overrides=None, accounts=None):
```

Comment:
It seems likely to cause confusion to have two methods of the same name (`generate_genesis_state`), especially in the same file. Maybe the other method could be called `generate_genesis_state_for_keys`.

Also, `num_accounts` here would be clearer as a variable name.

Reply:
:heavy_check_mark: